### PR TITLE
Update externals for ppc64 build

### PR DESCRIPTION
**Pull request by mrodozov**
Changes needed to build all externals on PowerPC (used the techlab-ppc64le-winstron-01 machine)
Only openloops remains to be fixed (failing with multiple fortran complains) and tfcompile (fails with similar complain to this one https://sft.its.cern.ch/jira/browse/ROOT-4385?focusedCommentId=24479&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel)

### CODE REVIEW

**Comment by mrodozov** [comment]
please test

File: vecgeom.spec [hunk 1]
```diff
@@ -5,15 +5,20 @@ BuildRequires: cmake gmake
 
 %define isamd64 %(case %{cmsplatf} in (*amd64*) echo 1 ;; (*) echo 0 ;; esac)
 %define isaarch64 %(case %{cmsplatf} in (*_aarch64_*) echo 1 ;; (*) echo 0 ;; esac)
+%define isppc64 %(case %{cmsplatf} in (*_ppc64le_*) echo 1 ;; (*) echo 0 ;; esac)
```
Comment (smuzaffar) [inline vecgeom.spec]: @mrodozov , can you please update the PR to use the following
`%ifarch ppc64le`

`%ifnarch ppc64le`

Comment (mrodozov) [reply vecgeom.spec]: Changed it everywhere and removed some defines

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
+externals
@fabiocos , this is needed for PowerPC builds

File: geneva.spec [hunk 2]
```diff
@@ -4,12 +4,20 @@ Source: git+https://stash.desy.de/scm/geneva/geneva-public.git?obj=master/%{real
 
 BuildRequires: cmake gmake
 
-Requires: python py2-setuptools py2-numpy gsl boost lhapdf hepmc openloops pythia8
+Requires: python py2-setuptools py2-numpy gsl boost lhapdf hepmc pythia8
+%ifnarch ppc64le
```
Comment (fabiocos) [inline geneva.spec:8]: not sure whether this will allow us to have a fully working version, but I understand that oterwise the build cannot proceed

**Comment by fabiocos** [comment]
+1

File: geneva.spec (hunk 2 above)
Comment (smuzaffar) [reply geneva.spec:8]: We do not build openloops for non-amd64 archs. We will change the above line to be `%ifarch x86_64` to make it consistent with sherpa, herwigpp and cmssw-tool-conf.

https://github.com/cms-sw/cmsdist/blob/8a4f8beb18943981d26b0b72b295e923a9b6223c/sherpa.spec#L12:L16
https://github.com/cms-sw/cmsdist/blob/ab6cf5ba4201f09658ff066c7d840f5a5e065023/herwigpp.spec#L16:L18
https://github.com/cms-sw/cmsdist/blob/95e43bbbf2ca2530091b8e44b19b4d4fa8aa1539/cmssw-tool-conf.spec#L173:L178

**Pull request merged**